appsec/tests/helper/client_test.cpp

```diff
@@ -74,9 +77,17 @@ int count_schemas(const std::map<std::string, std::string> &meta)
     return schemas;
 }
 
-network::client_init::request get_default_client_init_msg()
+network::client_init::request get_default_client_init_msg(
+    std::string rule_config = "valid")
 {
-    auto fn = create_sample_rules_ok();
+    std::string fn;
+    if (rule_config == "valid")
```

Comment:
Personally, I always wrap the content of conditionals with `{}`

Reply:
No problem I'll change it. I don't have a special preference 😅